openvdb/tree/InternalNode.h

```diff
@@ -302,6 +302,15 @@ class InternalNode
     void setOrigin(const Coord& origin) { mOrigin = origin; }
 
     Index32 leafCount() const;
+    void nodeCount(std::vector<Index32> &vec) const
```

Comment:
Probably best to follow convention here and put the implementation outside of the function declaration.